app/domain/authentication/authn_azure/verify_and_decode_azure_token.rb

```diff
@@ -0,0 +1,88 @@
+require 'jwt'
+
+module Authentication
+  module AuthnAzure
```

Comment:
shouldn't the name of the module be like the name of the file?

Reply:
No. The name of the command class is the name of the file.